File: Python/client4.py
import socket

def send_string_to_server(s, host='localhost', port=65432):
    with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as client_socket:
        client_socket.connect((host, port))
        client_socket.sendall(s.encode('utf-8'))
        print("Client 4: String sent to server:", s)

if __name__ == "__main__":
    send_string_to_server("multithreading example")

File: Java/Client4.java
import java.io.IOException;
import java.io.PrintWriter;
import java.net.Socket;

public class Client4 {
    private static final String SERVER_ADDRESS = "localhost";
    private static final int PORT = 65432;

    public static void main(String[] args) {
        try (Socket socket = new Socket(SERVER_ADDRESS, PORT);
             PrintWriter out = new PrintWriter(socket.getOutputStream(), true)) {

            out.println("multithreading example");
            System.out.println("Client 4: String sent to server: multithreading example");
        } catch (IOException e) {
            e.printStackTrace();
        }
    }
}
